pkg/controller/beat/metricbeat/config.go

```diff
@@ -37,36 +37,7 @@ var (
   - module: system
     period: 1m
     metricsets:
-    - filesystem
     - fsstat
-    processors:
-    - drop_event:
-        when:
-          regexp:
-            system:
-              filesystem:
-                mount_point: ^/(sys|cgroup|proc|dev|etc|host|lib)($|/)
-  - module: kubernetes
-    period: 10s
-    host: ${NODE_NAME}
-    hosts:
-    - https://${HOSTNAME}:10250
-    bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
-    ssl:
-      verification_mode: none
-    metricsets:
-    - node
-    - system
-    - pod
-    - container
-    - volume
-  - module: kubernetes
-    period: 10s
-    host: ${NODE_NAME}
-    hosts:
-    - https://${HOSTNAME}:10249
-    metricsets:
-    - proxy
```

Comment:
Can you clarify why we remove those? Is it moving us away from https://raw.githubusercontent.com/elastic/beats/7.7/deploy/kubernetes/metricbeat-kubernetes.yaml?